src/entities/Place/model.test.ts

```diff
@@ -499,7 +506,7 @@ describe(`findWithHotScenes`, () => {
           AND p.base_position IN (
             SELECT DISTINCT(base_position) FROM "place_positions" WHERE position IN ($1)
           )
-        ORDER BY p.like_rate DESC, p."deployed_at" desc
```

Comment:
If we don't remove like_rate, please test both cases

Reply:
same as prev comment